charts/external-dns/values.yaml

```diff
@@ -34,9 +34,9 @@ serviceAccount:
   # -- Annotations to add to the service account. Templates are allowed in both the key and the value. Example: `example.com/annotation/{{ .Values.nameOverride }}: {{ .Values.nameOverride }}`
   annotations: {}
   # -- (string) If this is set and `serviceAccount.create` is `true` this will be used for the created `ServiceAccount` name, if set and `serviceAccount.create` is `false` then this will define an existing `ServiceAccount` to use.
-  name:
+  name:  # @schema type:[string, null]; default: null
   # -- Set this to `false` to [opt out of API credential automounting](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting) for the `ServiceAccount`.
-  automountServiceAccountToken:
+  automountServiceAccountToken:  # @schema type:[boolean, null]; default: null
```

Comment:
Why not set the default to `true`, then there is no need to make it nullable.

Reply:
@marcofranssen this should be set to true. @ivankatliarchuk do you want to pick this and the other one up in your PR and set the value to `true` so we can get this merged?